### ISSUE TITLE
[Feature Request] Enable bonjour/zeroconf with avahi via web UI

### ISSUE THREAD

**Issue by idstein** [rockstor/rockstor-core#965]
As RockStor seeks to become a sleek and simple NAS solution, I would really welcome at least for the SOHO market to provide bonjour.

I think for now there's already a good integration with netatalk, for CIFS there needs to be only a new service definition and for NFS there could be additional options.

At least this would make it easier to handle within an IPv6 environment the addresses. Also a lot of mobile devices can easier detect the presence of a RockStor device in the near environment.

**Comment by schakrava** [comment]
Thanks for this idea @idstein . Have you tried implementing it manually on Rockstor? Though avahi is not listed as part of services in the System -> Services screen, we do use it for Samba.

Avahi for NFS does sound like a good idea. 

I am wondering if it can also be used to detect Rockstor service itself, so users can go to their Rockstor web-ui without having to find out what the IP is. @phillxnet , you may be interested in this as well.

**Comment by pusewicz** [comment]
Yeah, this can definitely be done, as FreeNAS already does it for you.
